caravel/assets/javascripts/explorev2/stores/store.js

```diff
@@ -2,39 +2,50 @@ export const initialState = {
   datasources: null,
   datasourceId: null,
   datasourceType: null,
-  vizType: null,
   timeColumnOpts: [],
-  timeColumn: null,
   timeGrainOpts: [],
   timeGrain: null,
-  since: null,
-  until: null,
   groupByColumnOpts: [],
-  groupByColumns: [],
   metricsOpts: [],
-  metrics: [],
   columnOpts: [],
-  columns: [],
   orderingOpts: [],
-  orderings: [],
-  timeStampFormat: 'smart_date',
-  rowLimit: 50000,
   searchBox: false,
   whereClause: '',
   havingClause: '',
   filters: [],
   filterColumnOpts: [],
+  viz: {
+    formData: {
+      sliceId: null,
+      vizType: null,
+      timeColumn: null,
+      timeGrain: null,
+      groupByColumns: [],
+      metrics: [],
+      since: null,
+      until: null,
+      having: null,
+      where: null,
+      columns: [],
+      orderings: [],
+      rowLimit: 50000,
```

Comment:
Thanks for bringing it up! I will issue a separate PR for setting all the default values in store based on what we have in forms.py :)